baal/active/dataset/pytorch_dataset.py

```diff
@@ -228,19 +178,19 @@ def label(self, index: Union[list, int], value: Optional[Any] = None) -> None:
 
     def reset_labeled(self):
```

Comment:
this is more of a style comment, since we used `labelling` and `labelled` every where with 2 `l`s, maybe rename to `reser_labelled` ?